### PR TITLE
Improve changelog

### PR DESCRIPTION
**Pull request by RichVanderwal**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,17 @@
...
+## [0.2.0] - 2020-10-28
+### Added
+- Support for metrics (#10)
+
+### Changed
+- Updated to use version 0.13.0 of the go.opentelemetry.io/otel packages. (#30)
+- Standardized CHANGELOG.md format.  When making changes to this project, add human-readable summaries of what you've done to the "Unreleased" section above. When creating a release, move that information into a new release section in this document. (#35)
+
+## [0.1.0] - 2019-12-31
 First release!
```
Comment (MrAlias) [inline CHANGELOG.md:17]:
```suggestion
First release!

[Unreleased]: https://github.com/newrelic/opentelemetry-exporter-go/compare/v0.2.0...HEAD
[0.2.0]: https://github.com/newrelic/opentelemetry-exporter-go/releases/tag/v0.2.0
[0.1.0]: https://github.com/newrelic/opentelemetry-exporter-go/releases/tag/v0.1.0
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,17 @@
-## ChangeLog
+# Changelog
+All notable changes to this project will be documented in this file.
 
-## 0.1.0
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
+## [Unreleased]
+
+## [0.2.0] - 2020-10-28
```
Comment (MrAlias) [inline CHANGELOG.md:8]: Should we release this as 0.13.0 to track upstream?

@jodeev thoughts?

Comment (RichVanderwal) [reply CHANGELOG.md:8]: Yep, sounds like we're going to be tracking upstream as well as possible, while making an honest attempt to preserve semver at the same time.

File: CHANGELOG.md (hunk 1 above)
Comment (RichVanderwal) [reply CHANGELOG.md:17]: Thanks for catching this! Do you think we should change the 0.2.0 entry to a "compare/v0.1.0...v0.2.0" just like the example at https://keepachangelog.com/en/1.0.0/ ?

Comment (MrAlias) [reply CHANGELOG.md:22]: Yeah, that sounds good as well. :+1:

**Review by MrAlias — APPROVED** [review]

**Pull request merged**